feat: rework Toast component (#1727)
WTTJ/welcome-ui: docs/pages/components/toast.mdx

## About

Notification component with [react-hot-toast](https://react-hot-toast.com/) and 2 components:

- Growl
- Snackbar

## Notifications

> Before use `Toast` component, you must to integrate `Notifications` component at root of your project. This component will contains all of your toasts in your DOM.

Reviewer: A ajouter dans le upgrade.mdx

> Before use `Toast` component, you must to integrate `Notifications` component at root of your project. This component will contains all of your toasts in your DOM.

Reviewer 2: ```suggestion
Before using the `Toast` component, you must add the `Notifications` component in your project root. This component will contain all your toasts.
```

Author: C'est mieux en effet 😅

}
```

> **Note:** The `useToast` hook is deprecated. Instead, you can directly use `toast` function.

Reviewer: à ne pas mettre ici je pense, mettre côté upgrade.mdx plutôt

## Options

The `toast` function takes a component to be displayed and an `options` object:

> - **position** : one of `top-left`, `top-center`, `top-right`, `bottom-left`, `bottom-center` (default) or `bottom-right`

Reviewer: A ajouter dans le upgrade.mdx

Author: Ha oui effectivement, j'ai omis de renseigner ce fichier!
